mesheryctl/internal/cli/root/system/model.go

```diff
@@ -356,6 +356,7 @@ mesheryctl exp model view [model-name]
 	RunE: func(cmd *cobra.Command, args []string) error {
 		if len(args) == 0 {
 			_ = cmd.Help()
+			return nil
```

Comment:
```suggestion
			return cmd.Help()
```

cmd.Help() will return `nil` so we can directly use it like above I guess.

Reply:
yes it can be

Reply:
@leecalcote I have to use `cmd.Help` func to show help message